core/src/main/scala/kafka/server/LogDirFailureChannel.scala

```diff
@@ -29,7 +32,7 @@ import java.util.concurrent.{ArrayBlockingQueue, ConcurrentHashMap}
  * can take the name of the new offline log directory out of the LogDirFailureChannel and handles the log failure properly.
```

Comment:
Not related to this PR, but it would be good to clarify that log dirs are added only once and they are deemed offline until a restart.